1_data/run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Download and Process SPICE2 Dataset from Zenodo
+mkdir -p data
+python get_data_spice2.py --data-dir "."
+#python get_data_spice2.py --data-dir "/dfs9/dmobley-lab/openff-bts"
+
+# Download and Process QCArchive Datasets
+## Singlpoint
+#python get_data_qca.py --datasets "OpenFF CX3-CX4 singlepoints v4.0" \
+#                       --dataset_type singlepoint \
+#                       --data_file ./data/singlepoint
+
+## Optimization
+#python get_data_qca.py --datasets "OpenFF Cresset Additional Coverage Optimizations v4.0" \
+#                       --dataset_type optimization \
+#                       --data_file ./data/optimization
+
+## Torsiondrive
+#python get_data_qca.py --datasets "OpenFF Additional Generated Guanidine Amidine Derivative and O-Linker TorsionDrives 4.0" \
+#                       --dataset_type torsiondrive \
+#                       --data_file ./data/torsion_data
```

Comment:
Are these the actual datasets you're using in the project or just an example of how to run it? Since this repo is also a record of the scientific work, please do comment when you're showing examples of useful scripts, vs processing data for actual use, for future confusion.

FYI, for this project I'm not sure combining QCA data and SPICE2 data for smee fitting will add value, since they're quite different levels of theory. If you wanted to extend the dataset, I think I'd look at OMol25 -- and they've actually drawn from and recomputed SPICE2, which is quite useful.

Reply:
They were examples since I was including the other script too. These were so that I could move forward in filtering/fitting with smaller datasets. I'll remove these since they are irrelevant now.

Reply:
Ok, thanks. In that case could you please just gitignore or add the qca script to a separate PR with the datasets you apply it to?? I'm mostly fine reviewing both but in the spirit of back-to-school it should mostly be one script per PR at least to start with.